### PR TITLE
[Bug 20046]  Interactive Tutorial: Make copy script / apply script into separate steps 

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: Toolset/palettes/tutorial/revtutorial.livecodescript [hunk 1]
```diff
@@ -3024,6 +3030,52 @@ on idePropertyChanged pObject
...
+end revTutorialCheckScript
+
+local sCheckScriptMsg
+constant kCheckBufferMS = 800
+on ideScriptEdited pScript, pObject
+   -- In order to mitigate against forgetting to press the
+   -- apply button, we subscribe to ideScriptEdited and check
+   -- to see if the script entered is correct. However, we 
+   -- don't want to interrupt the flow of typing, so add a
+   -- 10 millisecond buffer
```
Comment (runrevmark) [inline Toolset/palettes/tutorial/revtutorial.livecodescript]: You say 10 millisecond here, but the constant is 800...

Comment (livecodeali) [reply Toolset/palettes/tutorial/revtutorial.livecodescript]: Haha, oops. yes, originally went  for 10ms but it was way too short!

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 9f0be30

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 9f0be30e248089af59c72e1b33517945700fe58d

**Pull request merged**